packages/api/src/handlers/application-site/__tests__/application-site.spec.js

```diff
@@ -93,8 +88,7 @@ describe('The postApplicationSite handler', () => {
     expect(codeFunc).toHaveBeenCalledWith(201)
   })
 
-  it('returns a conflict and error if the user-application-site already exists', async () => {
-    models.users = { findByPk: jest.fn(async () => ({ dataValues: { id: 'bar' } })) }
+  it('returns a conflict and error if the application-site already exists', async () => {
```

Comment:
Couldn't we have multiple applications on one site?

Reply:
Yes indeed we can. The relation will allow it. Each site-application relation is unique though.